x/escrow/codec.pb.go

```diff
@@ -314,38 +221,10 @@ type UpdateEscrowPartiesMsg struct {
 	Recipient []byte `protobuf:"bytes,4,opt,name=recipient,proto3" json:"recipient,omitempty"`
 }
 
-func (m *UpdateEscrowPartiesMsg) Reset()         { *m = UpdateEscrowPartiesMsg{} }
-func (m *UpdateEscrowPartiesMsg) String() string { return proto.CompactTextString(m) }
-func (*UpdateEscrowPartiesMsg) ProtoMessage()    {}
-func (*UpdateEscrowPartiesMsg) Descriptor() ([]byte, []int) {
-	return fileDescriptor_36017ee554579951, []int{4}
-}
-func (m *UpdateEscrowPartiesMsg) XXX_Unmarshal(b []byte) error {
```

Comment:
There was a big issue with non-standard renderings here.

It was not (as all supppsed) an issue of protoc 3.4 vs 3.6 vs 3.7 but rather the version of gogoprotobuf that was being referenced... that in the vendor directory, or that in the GOPATH. It is nice to delete the repo from GOPATH to ensure we only import from vendor (or get a compile error).

If you use the most recent gogoprotobuf, as referenced in Gopkg.{toml,lock} you should have these XXX_ methods. I'm not sure why they added them, but they are a very noticeable side-effect of version drift

Reply:
Done!

I only had to install new compiler plugin binary. 

```
go get github.com/gogo/protobuf/protoc-gen-gogofaster
```